project/Dependencies.scala

```diff
@@ -111,7 +111,8 @@ object Dependencies {
   def excludeJakartaActivationApi(m: ModuleID): ModuleID = m.exclude("jakarta.activation", "jakarta.activation-api")
 
   val workspaceManager = excludeJakartaActivationApi("bio.terra" % "workspace-manager-client" % "0.16.0-SNAPSHOT")
-  val dataRepo = excludeJakartaActivationApi("bio.terra" % "datarepo-client" % "1.0.44-SNAPSHOT")
+  val dataRepo = excludeJakartaActivationApi("bio.terra" % "datarepo-client" % "1.41.0-SNAPSHOT")
+  val dataRepoJersey = "org.glassfish.jersey.inject" % "jersey-hk2" % "2.32"
```

Comment:
without the additional `jersey.inject` library, the upgraded data repo client encountered `Unexpected error: InjectionManagerFactory not found.` errors at runtime.

Reply:
is there a reason that the data repo client lib does not include all of its own dependencies?

Reply:
Good question! I raised the issue to them and they weren't aware of it.